<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveSettingsTestDialog</class>
 <widget class="QDialog" name="InteractiveSettingsTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>535</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="manifestTab">
      <attribute name="title">
       <string notr="true">Manifest</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QPlainTextEdit" name="manifestTextEdit">
         <property name="plainText">
          <string>{
    &quot;items&quot;: [
        {
            &quot;type&quot;: &quot;TextField&quot;,
            &quot;name&quot;: &quot;text&quot;,
            &quot;caption&quot;: &quot;Text Field&quot;,
            &quot;description&quot;: &quot;A text field&quot;,
            &quot;defaultValue&quot;: &quot;a text&quot;,
            &quot;value&quot;: &quot;a text&quot;
        },
        {
            &quot;type&quot;: &quot;GroupBox&quot;,
            &quot;name&quot;: &quot;group&quot;,
            &quot;caption&quot;: &quot;Group Box&quot;,
            &quot;items&quot;: [
                {
                    &quot;type&quot;: &quot;SpinBox&quot;,
                    &quot;name&quot;: &quot;number&quot;,
                    &quot;caption&quot;: &quot;Spin Box&quot;,
                    &quot;defaultValue&quot;: 42,
                    &quot;value&quot;: 42,
                    &quot;minValue&quot;: 0,
                    &quot;maxValue&quot;: 100
                },
                {
                    &quot;type&quot;: &quot;DoubleSpinBox&quot;,
                    &quot;name&quot;: &quot;double&quot;,
                    &quot;caption&quot;: &quot;Double Spin Box&quot;,
                    &quot;defaultValue&quot;: 3.14,
                    &quot;value&quot;: 3.14,
                    &quot;minValue&quot;: 0.0,
                    &quot;maxValue&quot;: 100.0
                },
                {
                    &quot;type&quot;: &quot;ComboBox&quot;,
                    &quot;name&quot;: &quot;combobox&quot;,
                    &quot;caption&quot;: &quot;Combo Box&quot;,
                    &quot;defaultValue&quot;: &quot;value2&quot;,
                    &quot;value&quot;: &quot;value2&quot;,
                    &quot;range&quot;: [&quot;value1&quot;, &quot;value2&quot;, &quot;value3&quot;]
                },
                {
                    &quot;type&quot;: &quot;CheckBox&quot;,
                    &quot;name&quot;: &quot;checkbox&quot;,
                    &quot;caption&quot;: &quot;Check Box&quot;,
                    &quot;defaultValue&quot;: true,
                    &quot;value&quot;: true
                },
                {
                    &quot;type&quot;: &quot;GroupBox&quot;,
                    &quot;name&quot;: &quot;buttons&quot;,
                    &quot;caption&quot;: &quot;Buttons&quot;,
                    &quot;items&quot;: [
                        {
                            &quot;type&quot;: &quot;Row&quot;,
                            &quot;name&quot;: &quot;row&quot;,
                            &quot;items&quot;: [
                                {
                                    &quot;type&quot;: &quot;Button&quot;,
                                    &quot;name&quot;: &quot;button1&quot;,
                                    &quot;caption&quot;: &quot;Button 1&quot;
                                },
                                {
                                    &quot;type&quot;: &quot;Button&quot;,
                                    &quot;name&quot;: &quot;button2&quot;,
                                    &quot;caption&quot;: &quot;Button 2&quot;
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string notr="true">Load Manifest</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string notr="true">Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="valuesTab">
      <attribute name="title">
       <string notr="true">Values</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="valuesTableWidget">
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
